package.json

```diff
@@ -11,21 +11,22 @@
   },
   "scripts": {
     "compile": "hardhat compile",
-    "lint-sol": "yarn solhint \"contracts/**/*.sol\"",
+    "lint-sol": "yarn solhint ./contracts/**/*.sol",
     "lint-ts": "yarn eslint ./test/** ./src/**",
-    "lint": "yarn lint-sol && yarn lint-ts",
+    "lint": "yarn lint-sol & yarn lint-ts",
     "clean": "hardhat clean",
     "build": "yarn run clean && yarn run compile",
     "typechain": "hardhat typechain",
     "test": "hardhat test",
     "semantic-release": "semantic-release --tag-format='v${version}-dev'",
-    "coverage": "hardhat coverage",
-    "check-coverage": "istanbul check-coverage --statements 90 --branches 87 --functions 89 --lines 90"
+    "check-coverage": "istanbul check-coverage --statements 90 --branches 87 --functions 89 --lines 90",
+    "coverage": "hardhat coverage && yarn check-coverage"
```

Comment:
why did you combine it? did you look into circleci yaml config? did you see how it works?
we need these to be different scripts because they are different jobs in the CI. since you've added && here, it will not go through fully if one of the scripts fails, breaking our flow in CircleCI.
we also need to be able to get local coverage files without checking coverage, so that we can actually see what we tests we are missing.
let's bring them back to be separate. and before making changes always ensure you are not breaking anything else by doing that.